pkg/claims/generator.go

```diff
@@ -0,0 +1,9 @@
+package claims
+
+import (
+	"fmt"
+)
+
+func GenerateName(vmName, networkName string) string {
```

Comment:
hm, I take "offense" at this name. At the end of the day, you're not generating anything. 

You're simply composing a name from others. 

And maybe a new file for this is a bit overkill ?... I won't insist on this second item though.

Reply:
I will rename it to `ComposeName` and move it to claims.go